zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -639,6 +639,39 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
 
+  /process-instances:
+    post:
+      tags:
+        - Process Instance
+      summary: Start process instance
+      description: |
+        Creates and starts an instance of the specified process. The process definition to use to
+        create the instance can be specified either using its unique key
+        (as returned by Deploy resources), or using the BPMN process ID and a version.
```

Comment:
Interesting insight, I wasn't aware at all. @pepopowitz, can you add this hint to the [dev guide](https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension)? Feel free to add any other documentation hints there as well. You can add me as a reviewer to the respective PR. When it's merged, I'll announce it in the respective Slack channels as well so devs are aware and we put even more emphasis on proper documentation.

Reply:
I will definitely do this. 

I'm also going to invest some more time next week into trying to hack our docs generator to be smarter about this, because it's already coming up more than I anticipated, and it has to be really annoying on the engineering side to have to worry about this detail.